bin/ocr-transform.sh

```diff
@@ -86,8 +87,7 @@ main () {
         [[ "$outfile" != '-' ]] &&  script_args=("${script_args[@]}" "-o:$outfile")
         exec_saxon "${script_args[@]}"
     else
-        script_args=("${script_args[@]}" "$infile")
-        script_args=("${script_args[@]}" "$outfile")
+        script_args=("$infile" "$outfile" "${script_args[@]}")
```

Comment:
This is reverse sorted then before.

Reply:
It is now sorted as described in [script/transform/README.md](https://github.com/UB-Mannheim/ocr-fileformat/blob/master/script/transform/README.md).

Reply:
It was wrong before, weird we never noticed this...